### ISSUE TITLE
Test enabling accessibility tests on Chromatic

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystemet#4509]
Our endusers might use [accessibility test in Chromatic](https://www.chromatic.com/docs/accessibility/) so test enabling this and see how our components pass.

Document what fails and how, if there are small technical elements we can fix or if accessibility interpretations can differ.

**Comment by mrosvik** [comment]
@mimarz turned this on now.
